index.js

```diff
@@ -0,0 +1,72 @@
+const express = require('express');
+const helmet = require('helmet');
+const cors = require('cors');
+const bcrypt = require('bcryptjs');
+const session = require('express-session');
+
+const db = require('./database/dbConfig.js');
+const Users = require('./users/users-modal.js');
+
+const server = express();
+
+const sessionConfig = {
+  name: 'notsession', // default is connect.sid
+  secret: 'nobody tosses a dwarf!',
+  cookie: {
+    maxAge: 1 * 24 * 60 * 60 * 1000,
+    secure: false, // only set cookies over https. Server will not send back a cookie over http.
+  }, // 1 day in milliseconds
+  httpOnly: true, // don't let JS code access cookies. Browser extensions run JS code on your browser!
+  resave: false,
+  saveUninitialized: false,
+};
+
+server.use(helmet());
+server.use(express.json());
+server.use(cors());
+server.use(session(sessionConfig));
+
+server.post('/api/register', (req, res) => {
+  let newUser = req.body;
+  const hash = bcrypt.hashSync(newUser.password, 16);
+  newUser.password = hash;
+
+  Users.addUser(newUser)
+       .then(saved => {
+         res.status(201).json(saved);
+       })
+       .catch(err => {
+         res.status(500).json(err);
+       });
+});
+
+server.post('/api/login', (req, res) => {
+  let { username, password } = req.body;
+
+  Users.findUser({username})
+       .first()
+       .then(user => {
+         if(user && bcrypt.compareSync(password, user.password)){
+           res.status(200).json({message: 'Logged in!'});
+         }
+         else{
+           res.status(401).json({message: 'You shall not PASS!'});
+         }
+       })
+       .catch(err => {
+         res.status(500).json(err);
+       });
+});
+
+server.get('/api/users', (req, res) => {
```

Comment:
Since all of your routes start with `/api`, you could use an express router to house them all within another route. Look into the docs here: https://expressjs.com/en/4x/api.html#router

Here's an example:
```javascript
const router = express.Router();

router.post('/register', (req, res) => { /* handle register logic */ });
router.post('/login', (req, res) => { /* handle login logic */ });
router.get('/users', (req, res) => { /* handle users logic */ });

server.use('/api', router);
```
This becomes a lot more useful once you start needing to split up your API into different directories and files, especially if there's a lot of crossover.

Reply:
Thanks for the feedback. I implemented the changes your recommended.